Scripts/HTMLDocsAutomation/HTMLDocsAutomation.py

```diff
@@ -287,7 +287,9 @@ def human_readable_example_to_html(hr_sample):
     hr_html: list = []
     while hr_sample:
         if hr_sample.startswith('#'):
-            title, hr_sample = hr_sample.split('\n', 1)
+            title = hr_sample
+            if '\n' in hr_sample:
```

Comment:
in case `hr_sample` does not include `\n` you should do 2 things:
1) `title` should consist with the value of `hr_sample` (which you did)
2) `hr_sample` should be empty (which you didn't do)